### PR TITLE
Work on search results UI

### PR DESCRIPTION
**Pull request by cmperez**

### CODE REVIEW

File: client/styles/style.css [hunk 1]
```diff
@@ -224,8 +288,8 @@ nav {
   background: #008c8c;
 }
 
-#x {
-  width: 200px;
+#trailing-space {
```
Comment (dduugg) [inline client/styles/style.css]: :joy:

**Comment by dduugg** [comment]
awesome work :+1: :heavy_multiplication_x: :100:

File: client/styles/style.css (hunk 1 above)
Comment (zindlerb) [inline client/styles/style.css]: :heart_eyes:

**Comment by zindlerb** [comment]
:+1:

**Pull request merged**